src/ResourceManager/Compute/Commands.Compute/Microsoft.Azure.Commands.Compute.dll-Help.xml

```diff
@@ -11738,6 +11738,241 @@ This cmdlet does not generate any output.</maml:para>
 <command:command xmlns:maml="http://schemas.microsoft.com/maml/2004/10" xmlns:command="http://schemas.microsoft.com/maml/dev/command/2004/10" xmlns:dev="http://schemas.microsoft.com/maml/dev/2004/10" xmlns:MSHelp="http://msdn.microsoft.com/mshelp">
 <!--Generated by PS Cmdlet Help Editor-->
 	<command:details>
+		<command:name>New-AzureVMSqlServerKeyVaultCredentialConfig</command:name>
```

Comment:
@hovsepm I have added the document in the help file in the new commit. Please review it. Thank you.